soc/v2/raspberrypi/Kconfig.soc

```diff
@@ -3,4 +3,4 @@
 # Copyright (c) 2021 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-source "soc/arm/rpi_pico/*/Kconfig.series"
+rsource "*/Kconfig.series"
```

Comment:
Probably the `Kconfig.series` should be sourced through the pure Zephyr Kconfig tree, and not the shared `Kconfig.soc` tree.

Like it's done here: https://github.com/zephyrproject-rtos/zephyr/blob/f78433a4c9f00bf847df6deae334d0eeaaaca0ff/soc/v2/nordic_nrf/Kconfig#L13

and then the pure SoC family / series definitions in the Kconfig.soc, like this:
https://github.com/zephyrproject-rtos/zephyr/blob/f78433a4c9f00bf847df6deae334d0eeaaaca0ff/soc/v2/nordic_nrf/Kconfig.soc